### PR TITLE
test: fix add mock call in CodeStar polling unit test

### PR DESCRIPTION
**Pull request by huanjani**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/aws/codestar/codestar_test.go [hunk 1]
```diff
@@ -27,6 +27,11 @@ func TestCodestar_WaitUntilStatusAvailable(t *testing.T) {
 		connection := &CodeStar{}
 		connectionARN := "mockConnectionARN"
 
+		ctrl := gomock.NewController(t)
+		defer ctrl.Finish()
+		m := mocks.NewMockapi(ctrl)
+		m.EXPECT().GetConnection(gomock.Any()).AnyTimes()
```
Comment (efekarakus) [inline internal/pkg/aws/codestar/codestar_test.go]: huh does this work without a `Return`?

**Review by iamhopaul123 — APPROVED** [review]

**Review by iamhopaul123 — COMMENTED** [review]
Oh maybe update the dependency as this PR trying to do https://github.com/aws/copilot-cli/pull/2085 so that we can make sure this fixes the unit test and won't block the dependency upgrade. After this PR is merged we can close https://github.com/aws/copilot-cli/pull/2085.

File: internal/pkg/aws/codestar/codestar_test.go (hunk 1 above)
Comment (Lou1415926) [reply internal/pkg/aws/codestar/codestar_test.go]: I wonder what would `aws.StringValue(output.Connection.ConnectionStatus)` be without a `Return` 🤔

Comment (efekarakus) [reply internal/pkg/aws/codestar/codestar_test.go]: 🤔 I just looked into the gomock library they return `nil` (the zero value of `interface{}`) by default for each return value! That's probably why it was working haha

**Review by efekarakus — APPROVED** [review]

**Pull request merged**